cloud-config/deploy-run-scripts/ami-elasticsearch.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# Setup elasticsearch data node config
+# - should not be called from frontend build
+echo -e "\n$ENCD_INSTALL_TAG $(basename $0)"
+
+# Check previous failure flag
+if [ -f "$encd_failed_flag" ]; then
+    echo -e "\n\t$ENCD_INSTALL_TAG $(basename $0) Skipping: encd_failed_flag exits"
+    exit 1
+fi
+if [ "$ENCD_BUILD_TYPE" == 'encd-es-build' ]; then
+    if [ ! -f "$encd_do_install_flag" ]; then
+        echo -e "\n\t$ENCD_INSTALL_TAG $(basename $0) Skipping ES AMI build. Set do install flag"
+        # For es builds we set the do install flag here since this is the main app
+        # We could have alternativily created an app-wrapper.sh for elasticsearch
+        touch "$encd_do_install_flag"
+        exit 0
+    fi
+fi
+echo -e "\n\t$ENCD_INSTALL_TAG $(basename $0) Running"
+
+# Script Below
+opts_src="$ENCD_CC_DIR/conf-es"
```

Comment:
where are the files in conf-es?    I feel like we need to be sure the indices.query.bool.max_clause_count: 8192 gets set.

Reply:
It is in there.  See /cloud-config/deploy-scripts/conf-es